packages/ra-ui-materialui/src/auth/LoginForm.tsx

```diff
@@ -29,33 +18,24 @@ interface FormData {
     password: string;
 }
 
-interface EnhancedProps
-    extends TranslationContextProps,
-        InjectedFormProps<FormData>,
-        WithStyles<typeof styles> {
-    isLoading: boolean;
-}
-
-const styles = (theme: Theme) =>
-    createStyles({
-        form: {
-            padding: '0 1em 1em 1em',
-        },
-        input: {
-            marginTop: '1em',
-        },
-        button: {
-            width: '100%',
-        },
-        icon: {
-            marginRight: theme.spacing(1),
-        },
-    });
+const useStyles = makeStyles((theme: Theme) => ({
+    form: {
+        padding: '0 1em 1em 1em',
+    },
+    input: {
+        marginTop: '1em',
+    },
+    button: {
+        width: '100%',
+    },
+    icon: {
+        marginRight: theme.spacing(1),
+    },
+}));
 
-// see http://redux-form.com/6.4.3/examples/material-ui/
 const renderInput = ({
```

Comment:
this is a Component, it should start with a capital. And the name doesn't feel right.